scully.config.js

```diff
@@ -6,8 +6,8 @@ require('./extraPlugin/voidPlugin');
 exports.config = {
   /** projectRoot is mandatory! */
   projectRoot: './projects/sampleBlog/src/app',
-  /** outFolder is where the static distribution files end up */
-  outFolder: './dist/static',
```

Comment:
as it is renamed: Probably a `ng-update` migration script is needed?

Reply:
Good catch. 
No, we are in Aplha, an ngUpdate script isn't needed, but the correct default is.